model/clusters_mgmt/v1/log_forwarding_groups_resource.model

```diff
@@ -0,0 +1,33 @@
+/*
+Copyright (c) 2025 Red Hat, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Manages the collection of log forwarder group versions.
+resource LogForwardingGroups {
+	// Retrieves the list of available log forwarder group versions.
+	method List {
```

Comment:
Any need for search or order? Mostly because it is kind of expected for the API, but I suppose clients will mostly all be fetching all anyways